ykrt/src/mt.rs

```diff
@@ -179,35 +179,35 @@ impl MTThread {
         &mut self,
         loc: Option<&Location<I>>,
         step_fn: S,
-        inputs: &mut I,
+        ctx: &mut I,
```

Comment:
Rename the type param?

Reply:
`I` does kind of work though, as it is the "[I]nterpreter context". Up to you.

Reply:
I actually did do that, but then realised that `I` neatly stands for "interpreter" and rolled back.

Reply:
k